flake.lock

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Are these Flake and Env files (`.envrc`, `flake.lock` and `flake.nix`) required for this PR? I just noticed the README mention that Nix Flakes is optional.

I suppose I have a small aesthetic interest in keeping the project root less cluttered but if it's required then it's fine.